src/assets/docker-entrypoint.sh

```diff
@@ -132,6 +134,25 @@ if (echo "${ACTIVEMQ_ARTEMIS_VERSION}" | grep -Eq  "(1.5\\.[3-5]|[^1]\\.[0-9]\\.
   performanceJournal
 fi
 
+###
+# Update dynamic env vars
+###
+# Add BROKER_CONFIGS env variable to startup options
+sed -i "/BROKER_CONFIGS/!s/^JAVA_ARGS=\"/JAVA_ARGS=\"\$BROKER_CONFIGS /g" $CONFIG_PATH/artemis.profile;
+
+# Loop through all BROKER_CONFIG_... and convert to java system properties
+env|grep -E "^BROKER_CONFIG_"|sed -e 's/BROKER_CONFIG_//g' >/tmp/brokerconfigs.txt
+while read -r config
+do
+  PARAM=${config%%=*}
+  PARAM_CAMEL_CASE=$(echo "$PARAM"|sed -r 's/./\L&/g; s/(^|-|_)(\w)/\U\2/g; s/./\L&/')
+  VALUE=${config#*=}
+  echo "PARAM=$PARAM_CAMEL_CASE VALUE=$VALUE"
```

Comment:
If this `echo` is intentional, could you print something more descriptive? "Broker config ${PARAM} translated to ${PARAM_CAMEL_CASE}".

Reply:
echo removed